pontoon/base/static/js/fluent_interface.js

```diff
@@ -228,7 +228,7 @@ var Pontoon = (function (my) {
           (translationAST && !self.isSupportedMessage(translationAST)) ||
           (!translationAST && !self.isSupportedMessage(entityAST))
         ) {
-          value = entity.key + ' = \n';
+          value = entity.key + ' = ';
```

Comment:
Why was this needed?

I'm asking because there's a bug in `fluent-syntax` 0.6.0 in parsing empty messages at EOF. See http://projectfluent.org/play/?gist=dcae41a5096d8fe24cc4b6fa1041534e. I wonder if perhaps you've run into it here as well?

Reply:
Pontoon only uses `.parseEntry()`, but I guess that doesn't matter?

I feel like this bug was indeed causing problems in Pontoon, because we previously inserted untranslated unsupported messages with trailing `\n` and then they didn't get parsed properly when detecting unsaved changes.

Reply:
Still, we seem to be good without trailing `\n` in 0.6.0 and also 0.4.2.

Reply:
The bug described in http://projectfluent.org/play/?gist=dcae41a5096d8fe24cc4b6fa1041534e was fixed in `fluent-syntax` 0.6.2. I updated the Playground to use it as well.